Update GCP user docs (#2355)
openshift/installer: docs/user/gcp/iam.md

# GCP Service Account

> In order to install OpenShift to your GCP project, the installer requires a service account, which provides authentication and authorization to access data in the Google APIs. You can use an existing IAM service account or create one following the steps outlined here:

Reviewer: ```suggestion
To install OpenShift to your GCP project, the installer requires a service account, which provides authentication and authorization to access data in the Google APIs. You can use an existing IAM service account that contains the required roles or create one by following these steps:
```

## Step 2: Assign Project Roles to your Service Account

> A service account needs to be granted permission for specific APIs in order to access the services used to created a cluster. You can assign the proper roles during or after the creation of a service account. The simplest approach to assigning roles would be to make the service account an Owner of the project, but that service account would then have complete control over the project, which would be a decided risk.

Reviewer: Is there a point in installation where you can remove the Owner role if you grant it here?

Reviewer 2: https://github.com/openshift/installer/blob/master/docs/user/gcp/postinstall.md
